tools/devops/automation/templates/main-stage.yml

```diff
@@ -195,7 +195,7 @@ stages:
           repositoryAlias: ${{ parameters.repositoryAlias }}
           commit: ${{ parameters.commit }}
 
-    - ${{ if eq(variables['Build.SourceBranch'], 'refs/heads/main') }}:
+    - ${{ if: and(eq(variables['Build.SourceBranch'], 'refs/heads/main'), eq(variables['Build.DefinitionName'], 'xamarin-macios-ci')) }}:
```

Comment:
This will work for the current name, won't if we rename it or move it. We should add a parameter with a default to true which will be set to false in the unified pipeline.

Reply:
@mandel-macaque  Could we add this for now so the localization can get turned back on and I can create an issue for this and make the changes when I get back?

Reply:
The PR is wrong, I won't land it as it is. I'll fix it myself.